ngraph/core/src/op/matmul.cpp

```diff
@@ -31,13 +31,27 @@ NGRAPH_SUPPRESS_DEPRECATED_START
 
 NGRAPH_RTTI_DEFINITION(op::MatMul, "MatMul", 0);
 
+op::MatMul::MatMul(const Output<Node>& A,
```

Comment:
Please do not change operations that are already introduced to supported opsets

Reply:
rolled back, thanks